### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by nekia**
We need to provide a good starting point for users.
Create a new README.md as a landing page for users.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -187,7 +187,7 @@ discover: $(BUILD_DIR)/bin/discover
 token: GO_LDFLAGS=-X $(pkgmap.$(@F))/metadata.Version=$(PROJECT_VERSION)
 token: $(BUILD_DIR)/bin/token
 
-blkvault-repo-docker: $(BUILD_DIR)/image/blkvault-repo/$(DUMMY)
+blkvault-repo-docker: $(BUILD_DIR)/images/blkvault-repo/$(DUMMY)
```
Comment (anand-jpa) [inline Makefile]: @nekia  I would like to remove all text containing "vault" and replace it with "archiver"

File: README-E2ETEST.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Hyperledger Fabric Block Archiving
+
+## Running end-to-end tests
+
+We using [behave](https://behave.readthedocs.io/en/latest/) python package which is BDD (behaviour-driven development) for e2e testing. uses tests written in a natural language style, backed up by Python code.
```
Comment (anand-jpa) [inline README-E2ETEST.md]: Change to "We are using [behave]...."
Also remove the full stop after "for e2e testing".
Instead use "and". That is, "for e2e testing and uses tests written..."

Comment (nekia) [reply README-E2ETEST.md]: Fixed it.

File: Makefile (hunk 1 above)
Comment (nekia) [reply Makefile]: Rename all 'vault' to 'archiver', including [fabric-block-archiving-testenv repository](https://github.com/nekia/fabric-block-archiving-testenv).

**Review by anand-jpa — APPROVED** [review]
Looks good to me.

**Pull request merged**